HW_Sem_1/HW_Task_3.py

```diff
@@ -0,0 +1,20 @@
+# Напишите программу, которая принимает на вход координаты точки (X и Y),
+# причём X ≠ 0 и Y ≠ 0 и выдаёт номер четверти плоскости
+# в которой находится эта точка (или на какой оси она находится).
+# Пример:
+# - x=34; y=-30 -> 4
+# - x=2; y=4-> 1
+# - x=-34; y=-30 -> 3
+
+x = int(input('Введите Х: '))
+y = int(input('Введите Y: '))
+if x > 0 and y > 0:
+    print(f'х={x}; y={y} => 1')
+else:
+    if x < 0 and y > 0:
+        print(f'х={x}; y={y} => 2')
+    else:
+        if x < 0 and y < 0:
```

Comment:
Это и верхние условия можно свернуть, например, так:
```suggestion
        if x < 0 > y:
```